docs/api/authentication.md

```diff
@@ -20,6 +21,9 @@ $ ssh git@git-server.com git-lfs-authenticate foo/bar.git download
     "Authorization": "RemoteAuth some-token"
   },
 
+  # optional, for expiring tokens, preferred over expires_at
+  "expires_in": 86400000000000
```

Comment:
Ha, nice catch -- I fixed those up in https://github.com/git-lfs/git-lfs/pull/2130/commits/2572163f392890d02ec02654d186b65343366bdd.